tests/test_solido.py

```diff
@@ -484,6 +506,45 @@ def deposit(lamports: int, expect_created_token_account: bool = False) -> None:
 assert result is None, f'Huh, perform-maintenance performed {result}'
 print('> There was nothing to do, as expected.')
 
+# Adding another validator
+print('\nAdd another validator')
+(validator_1, transaction_result) = add_validator(
+    'validator-account-key-1',
+    'validator-vote-account-key-1',
+)
+
+transaction_address = transaction_result['transaction_address']
+transaction_status = multisig(
+    'show-transaction',
+    '--multisig-program-id',
+    multisig_program_id,
+    '--solido-program-id',
+    solido_program_id,
+    '--transaction-address',
+    transaction_address,
+)
+approve_and_execute(transaction_address, test_addrs[0])
+
+result = solido(
+    'perform-maintenance',
+    '--solido-address',
+    solido_address,
+    '--solido-program-id',
+    solido_program_id,
+    keypair_path=maintainer.keypair_path,
+)
+
+del result['UnstakeFromActiveValidator']['from_stake_account']
+del result['UnstakeFromActiveValidator']['to_unstake_account']
+expected_result = {
+    'UnstakeFromActiveValidator': {
+        'validator_vote_account': validator.vote_account.pubkey,
+        'from_stake_seed': 0,
+        'to_unstake_seed': 0,
+        'amount': 1500000000,
```

Comment:
You can use numeric underscores in Python to make this large number more readable, i.e. `1_500_000_000`.